ISSUE_TEMPLATE

```diff
@@ -0,0 +1,17 @@
+### Context
+
+* [ ] **mapbox-gl-js version**: 
+* [ ] **browser version**: 
+* [ ] **operating system**: 
```

Comment:
Most bugs we encounter are not platform specific. I don't mind asking for browser / OS versions in a follow-up question if I can't reproduce. It is onerous to make users look this up without a strong need on our end.